ci/scenarios/edge/reroute/main.py

```diff
@@ -224,7 +223,7 @@ def test_reroute_fio(fio_bin_path, vpool, compute_client, cluster_info, api, dis
         for index in xrange(0, disk_amount):
             try:
                 vdisk_name = '{0}_vdisk{1}'.format(EdgeTester.TEST_NAME, str(index).zfill(4))
-                data_vdisk = VDiskHelper.get_vdisk_by_guid(VDiskSetup.create_vdisk(vdisk_name, vpool.name, EdgeTester.AMOUNT_TO_WRITE, std_2.storage_ip, api))
+                data_vdisk = VDiskHelper.get_vdisk_by_guid(VDiskSetup.create_vdisk(vdisk_name, vpool.name, EdgeTester.AMOUNT_TO_WRITE * 2, std_2.storage_ip, api))
```

Comment:
Disk size should be higher than the amount of data to write to be able to preallocate the space for fio